### DISCUSSION TITLE
colors.conf always empty?!

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1522] [Q&A] [answered]
Is it intended that the file `~/.config/hypr/themes/colors.conf` remains empty and never changes?

**Comment by kRHYME7** [comment, marked as answer]
yes. It's dynamic and used by wallbash

https://github.com/prasanthrangan/hyprdots/blob/main/Configs%2F.config%2Fhyde%2Fwallbash%2FWall-Dcol%2Fhypr.dcol


it will be purged if wallbash is disabled
